docs/source/developers/guide/documentation.rst

```diff
@@ -27,4 +27,65 @@
 
 **************************
 Helping with documentation
-**************************
\ No newline at end of file
+**************************
+
+
+A great way to contribute to the project is to improve
+documentation. 
+
+If you are an Arrow user and you found some docs to be
+incomplete or inaccurate, share your hard-earned knowledge
+with the rest of the community.
+
+If you haven't come across an error in the documentation
+it is hard to find one and going through the docs in search
+for corrections is not a good way to go. In this case you can
+search for an issue that is dealing with the documentation
+on JIRA.
+
+.. note::
+	When searching for JIRA issue that deals with documentation
```

Comment:
It seems your editor is configured to use tab characters for indentation, can you change it to only insert space characters?

(also, conventionally, the indentation size for `.rst` files is 3 spaces :-))

Reply:
Yeah, I got bitten by this yesterday! It should be correctly set in the new commit.